lib/commands/log.js

```diff
@@ -30,14 +37,17 @@ extensions.startLogCapture = async function () {
       realDeviceLogger: this.opts.realDeviceLogger,
       xcodeVersion: this.xcodeVersion,
     });
+    this.logs.safariConsole = new SafariConsoleLog(!!this.opts.showSafariConsoleLog);
   }
   try {
     await this.logs.syslog.startCapture();
   } catch (err) {
-    log.warn(`Continuing without capturing logs: ${err.message}`);
+    log.warn(`Continuing without capturing device logs: ${err.message}`);
     return false;
   }
   await this.logs.crashlog.startCapture();
+  await this.logs.safariConsole.startCapture();
```

Comment:
shouldn't we check if we are testing safari first or this logger should be always active?